packages/machine-extractor/src/toMachineConfig.ts

```diff
@@ -26,11 +31,15 @@ export interface ToMachineConfigOptions {
    * @default false
    */
   anonymizeInlineImplementations?: boolean;
+  /**
+   * If true, actions will be extracted as expressions
+   */
+  asExpressions?: boolean;
```

Comment:
All other alternatives are called like `*InlineImplementations`:
```suggestion
  stringifyInlineImplementations?: boolean;
```